Eastern-Research-Group/mywaterway
# Feature/714 climate tab outline

## Related Issues:
* [HMW-714](https://jira.epa.gov/browse/HMW-714)

## Main Changes:
* Added in a Climate tab, still a work in progress.
  * I wired up all of the layers that we already had.
  * I added in the Wild fires layer and wired it up, (not the historical part). 
  * For the date slider I ended up using the esri slider, because it allows vertical orientation. I thought about switching all date sliders over to esri, but wanted to get your opinion first. The one thing that react-ranger seems to do better, is it allows you drag the right handle past the left one and still have a range that makes sense. Esri just doesn't allow the handles to cross.
  * I plan on moving the initial layer configs to a config file. 

## Steps To Test:
1. Navigate to http://localhost:3000/community/020802011001/extreme-weather
2. Verify that layers load (note: a lot of the layers haven't been wired up)
3. Verify the switches make the layers visible/hidden
4. Verify that toggling layer visibilty from the layer list also toggles the corresponding switch
5. Block one of the services
6. Verify the status text, displays `N/A`
7. Verify the popups work on the wildfire layer

## TODO:
- [ ] Move initial layer configs to a config file on S3.
- [ ] Need to get input from EPA on what layers, if any, they want on by default.
- [ ] Wire up remaining switches/statuses.
